_translations/en.yml

```diff
@@ -207,6 +207,7 @@ en:
     walletblw: "Mobile wallet with Lightning Network support."
     walleteclairmobile: "Eclair Mobile is a next generation, Lightning-ready Bitcoin wallet. It can be used as a regular Bitcoin wallet, and can also connect to the Lightning Network for cheap and instant payments."
     walletwasabi: "Wasabi is an open-source, non-custodial, privacy focused Bitcoin wallet for Windows, Linux and Mac. Built-in Tor anonymity network, CoinJoin and coin control features."
+    walletunstoppable: "Unstoppable is one of the easiest and most advanced wallets for Bitcoin. It's built with best security and usability practices in mind. It is designed to communicate to the Bitcoin network directly without intermediaries."
```

Comment:
Please do not use superlatives or exclusive phrases in the wallet description, such as:

+ "one of the easiest and most advanced wallets"
+ "best security and usability practices"

Reply:
We've added more guidance on creating the description [here](https://github.com/bitcoin-dot-org/bitcoin.org/blob/master/docs/managing-wallets.md#adding-a-wallet) since you contributed your listing (yes, just days ago; no new requirements, just a clarification).

Reply:
> We've added more guidance on creating the description [here](https://github.com/bitcoin-dot-org/bitcoin.org/blob/master/docs/managing-wallets.md#adding-a-wallet) since you contributed your listing (yes, just days ago; no new requirements, just a clarification).

got it! agree on not using superlatives, was in a hurry when submitting it for the first time ;)

Reply:
Very good now, however you might be missing the word "a" in the English description.  Thanks.

Reply:
> Very good now, however you might be missing the word "a" in the English description. Thanks.

thanks) done